pennylane/tape/tape.py

```diff
@@ -249,6 +252,19 @@ class QuantumTape(AnnotatedQueue):
     >>> tape.num_params
     3
 
+    Iterating over the quantum circuit can be done by iterating over the tape
+    object:
+
+    >>> for op in tape:
+    ...     print(op)
+    RX(0.432, wires=[0])
+    RY(0.543, wires=[0])
+    CNOT(wires=[0, 'a'])
+    RX(0.133, wires=['a'])
+    expval(PauliZ(wires=[0]))
+
+    Tapes also support indexing.
```

Comment:
I might be worth to add an example here.

Reply:
Good idea, added!